### PR TITLE
doc: updated ceph monitor config options

### PR DESCRIPTION
**Pull request by falcon78921**
## Checklist
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

Executed `ceph-conf --dump-all` on a freshly installed v14.2.2 (nautilus)
cluster. Compared the global defaults to the keys/values specified in
`mon-config-ref.rst`. Checked `options.cc` to make sure the obsolete keys
are no longer used. 

If there's any errors or info missing, please let me know.

Fixes: https://tracker.ceph.com/issues/41516
Signed-off-by: James McClune <jmcclune@mcclunetechnologies.net>

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

File: doc/rados/configuration/mon-config-ref.rst [hunk 1]
```diff
@@ -322,7 +322,7 @@ by setting it in the ``[mon]`` section of the configuration file.
 :Description: Issue a ``HEALTH_WARN`` in cluster log when the monitor's data
               store goes over 15GB.
 :Type: Integer
-:Default: 15*1024*1024*1024*
+:Default: 16106127360
```
Comment (tchaikov) [inline doc/rados/configuration/mon-config-ref.rst]: i think `15*1024*1024*1024` is more readable than 16106127360. one can interpret as 15GB. but probably we can remove the trailing `*` , and quote it with "``".

File: doc/rados/configuration/mon-config-ref.rst [hunk 2]
```diff
@@ -398,15 +398,7 @@ by setting it in the ``[mon]`` section of the configuration file.
...
-:Default: ``0.66``
-
-
-``mon health data update interval``
-
-:Description: How often (in seconds) the monitor in quorum shares its health
-              status with its peers. (negative number disables it)
-:Type: Float
-:Default: ``60``
+:Default: ``0.660000``
```
Comment (tchaikov) [inline doc/rados/configuration/mon-config-ref.rst]: i don't think we need to use the default precision used by `printf()` when printing a float.

IMHO, `0.66` is more human-readable than `0.66000`.

File: doc/rados/configuration/mon-config-ref.rst [hunk 3]
```diff
@@ -1138,7 +1068,7 @@ Miscellaneous
               the stored checksums with the computed ones of all the stored
               keys.
 :Type: Integer
-:Default: 3600*24
+:Default: 86400
```
Comment (tchaikov) [inline doc/rados/configuration/mon-config-ref.rst]: for the same reason, `3600*24` is more readable than `86400` . as one is able to tell that it's 24 hours immediately without firing up `bc` =)

**Comment by falcon78921** [comment]
@tchaikov Made the necessary corrections, per your request. I agree that keeping the values human-readable is definitely a better idea. I was just copying what `ceph-conf` reported verbatim. I kept all of the float & double values to the hundredths place. I also added some spacing when descriptions were multiline. I don't think this will cause a problem with the reStructuredText rendering, but I'll find out. Thanks for reviewing Kefu! :)

**Comment by falcon78921** [comment]
jenkins render docs

**Pull request merged**